packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -177,93 +177,95 @@ class TextFormField extends FormField<String> {
     InputCounterWidgetBuilder buildCounter,
     ScrollPhysics scrollPhysics,
     Iterable<String> autofillHints,
+    AutoValidateMode autoValidateMode = AutoValidateMode.disabled,
```

Comment:
Actualy no. I would suggest `autovalidateMode`.

Reply:
@chunhtai do you have any suggestion on this?

Reply:
I am not sure what the question is. What is the contradictory here?

Reply:
I'm talking about the capitalization: autovalidateMode vs autoValidateMode

Reply:
I vote for autoValidateMode. The autovalidate we used before seems like a typo to me. We might as well fix it now if we are going to introduce a new property.

Reply:
Make sense.

Reply:
Sorry, to be specific, I think we just choose only one of the following for our capitalization to be consistent:

1. `AutoValidateMode autoValidateMode;`

or

2. `AutovalidateMode autovalidateMode;`

But we should not mix/match between these two options.  The original parameter is `autovalidate`, which seems to align with option 2.

Reply:
We seem to be the only framework that uses autovalidate while all other uses autoValidate. That makes me wonder if this is a typo or not.

Reply:
As @chunhtai said this seems a typo and since we are about to deprecate `autovalidate` parameter I think it is time to make things right.

Reply:
I'm on board with that 👍

Reply:
Though, just above I see that we also have `autofill` and not `autoFill`...

Reply:
autofill and autofocus seems like a validate word (by doing a google search). I can see people use it as a word, however, i couldn't find much usage of autovalidate as one word on the website. Maybe the word is more rare than autofill and autofocus, so people don't use it as much which make it not a word?

Reply:
talked with @justinmc offline. We are both leaning toward using autovalidate now because all other auto- properties use lower case. It is not correct to defer the decision to the internet usages, because internet is known for its inconsistency, and that will make things complex. We should prefer a more unified API across the flutter app.

Reply:
Great.So I will rename it to `autovalidateMode`. Is that right?

Reply:
That's right, `autovalidateMode` and `AutovalidateMode`.